### DISCUSSION TITLE
Limits and usage rules

### DISCUSSION THREAD

**Discussion by LuisPaGarcia** [pockethost/pockethost#257] [Q&A] [answered]
Hello,

I'm starting using pockethost, and is awesome! 

I didn't found the limits, pricing or limitations of usage. Where I can find it?

I mean:
- API calls per period of time.
- Max size of the DB.

With this clarity I can have more confidence of continue using Pockethost, with the clear amount of the quotas that I have available for my app.

**Comment by benallfree** [comment, marked as answer]
Hello, that info is in the FAQ https://pockethost.gitbook.io/manual/overview/faq

There are no usage limits for now.
